src/styles/blocks/header.scss

```diff
@@ -0,0 +1,80 @@
+@import '../utils/variables';
+
+.header {
+  width: 100vw;
```

Comment:
Using `100vw` for width can cause horizontal scrolling issues due to the scrollbar width. Consider using `100%` instead to avoid this potential issue.